config/application.rb

```diff
@@ -27,9 +27,13 @@ class Application < Rails::Application
       "I18n::InvalidLocale" => :not_found
     )
 
-    # Rails handles ParseError by default, but it is occurring on SQS and I need more details about why,
-    # so turn it off.
-    ActionDispatch::ExceptionWrapper.rescue_responses.delete("ActionDispatch::ParamsParser::ParseError")
+    if defined? ActiveElasticJob
```

Comment:
http://guides.rubyonrails.org/initialization.html#config-application-rb the application.rb comes before all the other initializers, so I don't think we need to.